agents/agents-core/src/jvmTest/kotlin/ai/koog/agents/core/system/feature/DebuggerTest.kt

```diff
@@ -292,6 +292,9 @@ class DebuggerTest {
                 val startGraphNode = StrategyEventGraphNode(id = "__start__", name = "__start__")
                 val finishGraphNode = StrategyEventGraphNode(id = "__finish__", name = "__finish__")
 
+                val callIds = clientEventsCollector.collectedEvents.filterIsInstance<LLMCallStartingEvent>().map { it.callId }
+                require(callIds.size == 2) { "Expected 2 LLMCallStartingEvent, got ${callIds.size}" }
```

Comment:
Here and in other tests: please use `assert...` functions instead of `require`, e.g. `assertEquals`, it would slightly improve the style and communicate clearly that we are testing the condition